src/physics/em/detail/CombinedBremInteractor.i.hh

```diff
@@ -33,13 +33,11 @@ CombinedBremInteractor::CombinedBremInteractor(
     StackAllocator<Secondary>&   allocate,
     const MaterialView&          material,
     const ElementComponentId&    elcomp_id)
-    : shared_(shared)
-    , inc_energy_(particle.energy())
+    : inc_energy_(particle.energy())
```

Comment:
I think these files were changed by mistake as part of a rebase/merge: I'll pause reviewing here.

Reply:
Actually that was me. After doing `git merge` (at 6ea7078), I unexpectedly got a bunch of `variable not used` errors on my side, so I went ahead a updated them.

Reply:
Oh ok. I remember that comment. Can you move those to another MR?

Reply:
Sure can. I'll open it up asap, so I can merge those changes into this PR and avoid missing something else that otherwise would have been caught by the `-Wall -Wextra -Werror -pedantic` cmake flags